idaes/core/base/property_set.py

```diff
@@ -755,6 +756,11 @@ class StandardPropertySet(PropertySetBase):
         doc="Specific Gibbs Energy (Molar Basis)",
         units="ENERGY_MOLE",
     )
+    ionic_strength = PropertyMetadata(
```

Comment:
Should this be defined in the electrolyte properties instead?

Reply:
Yes, I think that makes sense.  I'll move them.